templates/security/cis.html

```diff
@@ -0,0 +1,145 @@
+{% extends "security/base_security.html" %}
+
+
+{% block title %}CIS Benchmark on Ubuntu{% endblock %}
+{% block meta_description %}Technical details on all of the security certifications for Ubuntu Advantage subscribers.{% endblock %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1bSv8lV9BJoBYh5yog2eYKtvNMitKZetSSBQ3k2Cu5Cw/edit#heading=h.335zx3wyom0b#{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip--suru-topped is-bordered">
+  <div class="row u-equal-height">
+    <div class="col-8">
+      <h1>CIS Benchmark on Ubuntu</h1>
+      <p class="p-heading--four"><i>Comply with the most widely accepted baseline</i></p>
+      <p>The Center for Internet Security (CIS) is a non-profit organisation with a mission to “make the connected world a safer place by developing, validating, and promoting timely best practice solutions against pervasive cyber threats”. CIS uses a consensus process to release benchmarks to safeguard organisations against cyber attacks. The consensus review process consists of subject matter experts who provide perspective on different backgrounds like audit and compliance, security research, consulting and software development. The benchmarks are considered a necessary complement in the implementation of a cybersecurity framework, and are the most widely accepted Industry benchmarks to harden a system today.</p>
+      <p>The CIS benchmark has hundreds of configuration recommendations, so hardening and auditing a system manually can be very tedious. To drastically improve this process for enterprises, Canonical provides tooling for audit and automated compliance with the CIS benchmarks with <a href="/advantage">Ubuntu Advantage</a> and <a href="/public-cloud">Ubuntu Pro</a>.</p>
+      <a href="/support/get-in-touch" class="p-button--positive">Access the tooling for audit and automated CIS compliance</a>
```

Comment:
Can you add this `js-invoke-modal` class to the button, it should be a pop up modal when clicked. 
```suggestion
      <a href="/support/get-in-touch" class="p-button--positive js-invoke-modal">Access the tooling for audit and automated CIS compliance</a>
```